src/PostgREST/Error.hs

```diff
@@ -289,6 +291,9 @@ instance JSON.ToJSON Error where
   toJSON (JwtTokenInvalid message) = JSON.object [
     "message" .= (message :: Text)]
   toJSON NotFound = JSON.object []
+  toJSON RpcNotFound = JSON.object [
+    "hint"    .= ("No function matches the given name and argument types. If a new function was created in the database with this name, try updating the schema cache." :: Text),
```

Comment:
Hm. The `hint` should serve as a guide for how to clear the error.  So the `No function matches the given name and arguments` message doesn't belong to the `hint`. It should go in `error`.

Reply:
Right now, postgrest shows the `No function matches the given name and arguments` error in the hint when a function is not found (the database raises this error). But I agree that it should be part of the message.